archtitus.sh

```diff
@@ -18,10 +18,9 @@ echo -ne "
     bash startup.sh
     source setup.conf
     bash 0-preinstall.sh
-    arch-chroot /mnt /root/$SCRIPTHOME/1-setup.sh
-    source /mnt/root/$SCRIPTHOME/install.conf
```

Comment:
@fhanrath Was it intended to remove source install.conf reference in this PR?

Reply:
Yes, afaik the sourced parameters are not available inside the `arch-chroot` environment anyways.
And in this branch the `startup.sh` script gets called which would overwrite the params of the `install.conf`.

If I'm wrong, I'll gladly restore the `source /mnt/root/$SCRIPTHOME/install.conf`.